mongo/mongo.go

```diff
@@ -96,3 +96,15 @@ func (m *Mongo) GetCacheTime(ctx context.Context, id string) (*models.CacheTime,
 	}
 	return &result, nil
 }
+
+// UpsertCacheTime adds or overrides an existing cache time
+func (m *Mongo) UpsertCacheTime(ctx context.Context, cacheTime *models.CacheTime, id string) (err error) {
```

Comment:
I think you can get rid of the `id` parameter, since it's already part of `cacheTime`.

Reply:
Removed 👍